README.md

```diff
@@ -264,28 +143,16 @@ I would like to thank...
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- FAQ (optional) -->
 
 ## ❓ FAQ <a name="faq"></a>
 
-> Add at least 2 questions new developers would ask when they decide to use your project.
-
-- **[Question_1]**
-
-  - [Answer_1]
-
-- **[Question_2]**
-
-  - [Answer_2]
+N/A
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
 
```

Comment:
- [ ] Your MIT link leads to a 404 error page. I'll suggest you add a license file to your project and link it appropriately. :wave:  

- - Screenshot reference:
![Selection_012](https://github.com/SaveryIV/vet-clinic/assets/107632780/294def38-63cd-41bd-b74f-a27c776e43ab)

## When naming your branch or pull request title, you are supposed to give it a descriptive name that's tailored to what the branch and the requirements you are working on are all about. The bench name and pull request title `Day-3` are neither descriptive nor clear on what this branch contains. 
## I suggest you change the pull request title to something more detailed like `Create and query multiple databases` and in your subsequent project, try to give your branch a better descriptive name like `Query-multiple-tables`